test/karma.conf.js

```diff
@@ -94,7 +101,7 @@ module.exports = function(config) {
     // - Safari (only Mac)
     // - PhantomJS
     // - IE (only Windows)
-    browsers: ['Chrome', 'Firefox'],
+    browsers: ['Firefox'],
```

Comment:
Should I change the code to pass the --browsers parameter from run_each.sh only when the BROWSERS_TO_TEST environment variable is set?